testdata/catalogs/test-catalog/catalog.yaml

```
@@ -74,7 +74,7 @@ entries:
 schema: olm.bundle
 name: package-with-webhooks.1.0.0
 package: package-with-webhooks
-image: localhost/testdata/bundles/registry-v1/package-with-webhooks:v1.0.0
+image: docker-registry.operator-controller-e2e.svc.cluster.local:5000/bundles/registry-v1/prometheus-with-webhooks.v1.0.0
```

Should this be a `:` instead of a `.` in front of version?
```suggestion
image: docker-registry.operator-controller-e2e.svc.cluster.local:5000/bundles/registry-v1/prometheus-with-webhooks:v1.0.0
```